### ISSUE TITLE
[UX Friction] Scanning a complete document

### ISSUE THREAD

**Issue by CarolinaCst** [owncloud/ios-app#758]
Problems focusing a complete document to scan

### Steps to reproduce
1. Open the iOS App 1.2.3
2. Users press "plus button" and click on "Scan document"

### Expected behavior
Scan the entire page

### Actual behavior
Sometimes a blue box appear moving faster unable to focus on the complete document to scan it

iOS 1.3.2 

### Tell us what could be improved:
Scan the complete document without dragging to adjust the hole page
If the camera is having troubles to focus the document an advice could appear:

- Place your mobile around 20cm to scan the entire document

![IMG_3298](https://user-images.githubusercontent.com/60343863/88533678-f9f22a80-d006-11ea-819d-f0dfdbba768e.PNG)
![IMG_3300](https://user-images.githubusercontent.com/60343863/88533732-0a0a0a00-d007-11ea-8050-de434b1c6422.PNG)
![IMG_3301](https://user-images.githubusercontent.com/60343863/88533778-19895300-d007-11ea-8602-ef5a2ad8b221.PNG)

Videos:
Participant_G: 5:37Min/  7:10Min/ 7:35Min
Participant_FA3: 0:21Min
Participant_AB: 0:34 Min

**Comment by mneuwert** [comment]
@CarolinaCst We use a black box document scanner solution from iOS Vision framework. And this UI is provided by the system component. Nothing we could easily change. In case system page edges detection algorithm is having issues due e.g. bad lighting conditions or lacking contrastful separation from the page background, the only workaround would be to press "Shutter" button manually and to drag markers to corners of the page...

As @michaelstingl already mentioned in #757 we could show a short tutorial to the user or something similar.

**Comment by CarolinaCst** [comment]
Yes, then I think a tutorial/ Onboarding would be an option then!
